src/utilities/eprint.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 28 11:51:24 2018
+
+@author: mmacferrin
+
+eprint() a quick utility function for printing to stderr instead of stdout.
+"""
+
+from __future__ import print_function
+import sys
+
+def eprint(*args, **kwargs):
```

Comment:
@mmacferrin , 

why do we need eprint, and where do we use it exactly, thanks?

Reply:
I agree here with @AymenFJA . This function just import `print` from future. It makes more sense to have the `from...` part where it is used.

It will also make the transition to python3 later in the year easier.

Reply:
I was using it from the asift_executable.py code in Phase 2 (where outputs to stdout and stderr were being handled a bit differently). But if it's easier to just use the print() function in Python 3 (agreed on that) and you don't feel this function is needed at all, feel free to drop it, no worries from me. This could easily be done after the merge, for instance.

Reply:
Yes, I agree! Thank you for clarifying